tensorflow_datasets/core/api_utils.py

```diff
@@ -45,16 +67,24 @@ def disallow_positional_args(wrapped=None, allowed=None):
     return functools.partial(disallow_positional_args, allowed=allowed)
 
   @wrapt.decorator
-  def disallow_positional_args_dec(fn, instance, args, kwargs):
+  def disallow_positional_args_dec(fn: Callable[[], Any],
+                                   instance: Optional[object],
+                                   args: Sized, kwargs: Iterable) -> object:
```

Comment:
Why `Sized` ? `Iterable` ? I think it should be `*args: Any, **kwargs: Any`

Reply:
Done